spec.html

```diff
@@ -4200,46 +4200,62 @@ <h1>UpdateEmpty ( _completionRecord_, _value_ )</h1>
-      <p>A <dfn id="super-reference">Super Reference</dfn> is a Reference that is used to represent a name binding that was expressed using the super keyword. A Super Reference has an additional thisValue component, and its base value component will never be an Environment Record.</p>
-      <p>The following abstract operations are used in this specification to operate on references:</p>
-
-      <emu-clause id="sec-getbase" aoid="GetBase" oldids="ao-getbase">
-        <h1>GetBase ( _V_ )</h1>
-        <p>The abstract operation GetBase takes argument _V_. It performs the following steps when called:</p>
-        <emu-alg>
-          1. Assert: Type(_V_) is Reference.
-          1. Return the base value component of _V_.
-        </emu-alg>
-      </emu-clause>
+      <p>The <dfn>Reference</dfn> type is a Record used to explain the behaviour of such operators as `delete`, `typeof`, the assignment operators, the `super` keyword and other language features. For example, the left-hand operand of an assignment is expected to produce a Reference.</p>
+      <p>A Reference is a resolved name or property binding. Values of the Reference type are Record values whose fields are defined by <emu-xref href="#table-reference-fields"></emu-xref>.</p>
 
-      <emu-clause id="sec-getreferencedname" aoid="GetReferencedName" oldids="ao-getreferencedname">
-        <h1>GetReferencedName ( _V_ )</h1>
-        <p>The abstract operation GetReferencedName takes argument _V_. It performs the following steps when called:</p>
-        <emu-alg>
-          1. Assert: Type(_V_) is Reference.
-          1. Return the referenced name component of _V_.
-        </emu-alg>
-      </emu-clause>
+      <emu-table id="table-reference-fields" caption="Reference Fields">
+        <table>
+          <tbody>
+            <tr>
+              <th>Field Name</th>
+              <th>Value</th>
+              <th>Meaning</th>
+            </tr>
+            <tr>
+              <td>[[Base]]</td>
+              <td>
+                One of:
+                <ul>
+                  <li>
+                    ECMAScript language value except *undefined* and *null*;
+                  </li>
+                  <li>
+                    Environment Record;
+                  </li>
+                  <li>
+                    ~empty~.
+                  </li>
+                </ul>
+              </td>
+              <td>~empty~ value indicates that the Reference could not be resolved to a binding.</td>
+            </tr>
+            <tr>
+              <td>[[ReferencedName]]</td>
```

Comment:
Can we give these IDs or something so that links to `GetReferencedName`, for example, don't break?

Reply:
Resolved in 8145f01.